src/main/java/frc/robot/subsystems/Climbers.java

```diff
@@ -0,0 +1,128 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ClimberConstants;
+import frc.robot.Constants.DriveConstants;
+
+public class Climbers extends SubsystemBase {
+  
+  private final CANSparkMax m_leftClimber;
+  private final CANSparkMax m_rightClimber;
+
+  private final CANEncoder m_leftEncoder;
+  private final CANEncoder m_rightEncoder;
+  
+  private final Solenoid frictionBrake;
```

Comment:
Per WPI style guides this should start with `m_` since the value is constant.
```suggestion
  private final Solenoid m_frictionBrake;
```

Reply:
Yeah thanks! Missed that